tests/partition/test_clobbering.py

```diff
@@ -0,0 +1,21 @@
+from partition.__main__ import clobbering
```

Comment:
The `clobbering` function no longer exists, so this test is failing with an error.